maintain.go

```diff
@@ -394,10 +394,12 @@ func (certCache *Cache) updateOCSPStaples(ctx context.Context) {
 	// These write locks should be brief since we have all the info we need now.
 	for certKey, update := range updated {
 		certCache.mu.Lock()
-		cert := certCache.cache[certKey]
-		cert.ocsp = update.parsed
-		cert.Certificate.OCSPStaple = update.rawBytes
-		certCache.cache[certKey] = cert
+		cert, found := certCache.cache[certKey]
+		if found {
```

Comment:
```suggestion
		if cert, ok := certCache.cache[certKey]; ok {
```

Just a little more conventional Go style here :)